addon/helpers/build-secondary-nav-links.js

```diff
@@ -0,0 +1,83 @@
+import Ember from 'ember';
+import config from 'ember-get-config';
+
+/**
+ * @module ember-osf
+ * @submodule helpers
+ */
+
+/**
+ * Returns secondary links corresponding to a given service
+ *
+ * @class buildSecondaryNavLinks
+ * @param {String} currentService name of current service (HOME, PREPRINTS, REGISTRIES, or MEETINGS)
+ * @param {Object} session Injected session
+ * @return {Array} Returns array of secondary link information
+ */
+export function buildSecondaryNavLinks(params/*, hash*/) {
+    const currentService = params[0].toUpperCase();
+    const session = params[1];
+    const osfUrl = config.OSF.url;
+    let links = Ember.Object.create({
+        HOME: [
+            {
+                name: `${session.isAuthenticated ? 'My Projects' : 'Browse'}`,
+                href: `${session.isAuthenticated ? `${osfUrl}myprojects/` : `${osfUrl}explore/activity/`}`
```

Comment:
Perhaps not now, but for the future: I'm starting to notice a number of places throughout the codebase that use service links. (eg navbar, footer) 

In the future we might want to consider centralizing those URL definitions in one place, especially for services that don't depend on whether or not the user is logged in.